Cargo.toml

```
@@ -26,14 +26,15 @@ clock = ["std", "winapi", "iana-time-zone", "android-tzdata"]
 oldtime = []
 wasmbind = ["wasm-bindgen", "js-sys"]
 unstable-locales = ["pure-rust-locales"]
+rkyv-validation = ["rkyv/validation"]
 __internal_bench = []
 
 [dependencies]
 num-traits = { version = "0.2", default-features = false }
 rustc-serialize = { version = "0.3.20", optional = true }
 serde = { version = "1.0.99", default-features = false, optional = true }
 pure-rust-locales = { version = "0.7", optional = true }
-rkyv = { version = "0.7", optional = true }
+rkyv = { version = "0.7.42", optional = true }
```

```suggestion
rkyv = { version = "0.7.41", optional = true }
```

I hope that by setting `0.7.41` as the minimum version and with https://github.com/rkyv/bytecheck/pull/33 that we can get things working with our MSRV.

---

I'm curious why `0.7.42` was not able to be found 🤔 🤔